vnext/Microsoft.ReactNative/Fabric/ComponentViewRegistry.cpp

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "pch.h"
+
+#include "ComponentViewRegistry.h"
+
+#pragma warning(push)
+#pragma warning(disable : 4244 4305)
+#include <react/renderer/components/scrollview/ScrollViewShadowNode.h>
+#include <react/renderer/components/text/ParagraphShadowNode.h>
+#include <react/renderer/components/text/TextShadowNode.h>
+#include <react/renderer/components/view/ViewShadowNode.h>
+#pragma warning(pop)
+
+#include "ParagraphComponentView.h"
+#include "ScrollViewComponentView.h"
+#include "TextComponentView.h"
+#include "ViewComponentView.h"
+
+namespace Microsoft::ReactNative {
+
+ComponentViewDescriptor const &ComponentViewRegistry::dequeueComponentViewWithComponentHandle(
```

Comment:
Confused about the naming here. Specifically the `dequeue` bit.

Reply:
From reading `enqueueComponentViewWithComponentHandle` I think these might be reversed?

Reply:
Consider adding some docs to headers for methods/classes like this. Would make it easier to understand intent.

Reply:
These align with the same functionality + names in core.